### ISSUE TITLE
BUG: Opening/closing rune pouch configure interface bugs the plugin

### ISSUE THREAD

**Issue by fuhsaz** [dillydill123/inventory-setups#252]
It doesn't happen every time, but frequently opening/closing the configure rune pouch interface makes the plugin look weird.
![327134667-84fc2706-a4bd-4adf-98c0-0e03d855db52](https://github.com/dillydill123/inventory-setups/assets/15990644/bd29aae8-fb93-427b-b2b7-d3afa756217c)

**Comment by fuhsaz** [comment]
This also sometimes happens when switching between "withdraw as item" and "withdraw as note".

**Comment by dillydill123** [comment]
you're seeing weird behavior probably because of bank tag layouts. Additionally, "withdraw as note" is a known issue with bank tags. Might be the same issue with the rune pouch https://github.com/dillydill123/inventory-setups/issues/29

**Comment by BattleFlyNate** [comment]
> you're seeing weird behavior probably because of bank tag layouts. Additionally, "withdraw as note" is a known issue with bank tags. Might be the same issue with the rune pouch #29

Not entirely bank tag layouts fault. This is still an issue if clicking a rune pouch with a setup open. It will simply close the tab in the bank window. Not sure if fixing that would fixing the issue with bank tag layouts too, but it's probably exacerbating the issue.

**Comment by IsaacGraham** [comment]
I've had this issue consistently but as of the recent update it seems to not be happening anymore?

Cannot confirm based on single person sample size but does seem to be working fine now.

**Comment by dillydill123** [comment]
Yeah it’s fixed because of the migration to core bank tags in inventory setups
